src/runmode-pfring.c

```diff
@@ -123,7 +123,11 @@ static void *OldParsePfringConfig(const char *iface)
         pfconf->threads = 1;
     } else {
         if (threadsstr != NULL) {
-            pfconf->threads = atoi(threadsstr);
+            if (ByteExtractStringUint16(&pfconf->threads, 10, 0, (const char *)threadsstr) < 0) {
```

Comment:
Cast on `threadsstr` isn't needed.

Please check the other calls to `ByteExtract.*(` -- I suspect there are many cases where the cast isn't necessary.